### PR TITLE
UI-174 - Use custom table in opc-ua page

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

Related to: #174

### CODE REVIEW

File: src/app/pages/services/opcua/add/opcua.add.component.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  styleUrls: ['./opcua.add.component.scss'],
+})
+export class OpcuaAddComponent {
+  @Input() formData: OpcuaTableRow = {
+    name: '',
+    serverURI: '',
+    nodeID: '',
+  };
+  @Input() action: string = '';
+  constructor(
```
Comment (darkodraskovic) [inline src/app/pages/services/opcua/add/opcua.add.component.ts:20]: Separate this with a blank line from the `action` declaration.

File: src/app/pages/services/opcua/add/opcua.add.component.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    const nodes = [{
+      name: this.formData.name,
+      serverURI: this.formData.serverURI,
+      nodeID: this.formData.nodeID,
+    }];
+
+    if (this.action === 'Create') {
+      this.opcuaService.addNodes(this.formData.serverURI, nodes).subscribe(
+        resp => {
+          this.notificationsService.success('OPC-UA device successfully created', '');
```
Comment (darkodraskovic) [inline src/app/pages/services/opcua/add/opcua.add.component.ts:45]: Maybe we should move this notifs in the `opcuaService` (a remark for some future PR).

**Review by blokovi — DISMISSED** [review]

File: src/app/pages/services/opcua/add/opcua.add.component.ts (hunk 2 above)
Comment (manuio) [reply src/app/pages/services/opcua/add/opcua.add.component.ts:45]: I agree

Comment (manuio) [reply src/app/pages/services/opcua/add/opcua.add.component.ts]: done!

**Review by darkodraskovic — APPROVED** [review]

**Review by blokovi — APPROVED** [review]

**Pull request merged**